Azure-Samples/azure-search-openai-demo
# 6 as a user i want to delete my chats

## Purpose
Adds delete confirmation capability for conversastion history list. 

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Deploy repo.
Have a conversation. 
It should be saved.
Delete it.
You should see a modal asking for confirmation. Confirming will delete; canceling will close modal.
